scalableminds/webknossos
# Add frontpage for tracingstore + datastore

Standalone Datastores and Tracingstores replied with Play’s default “action not found” page on `/` or `/index.html` requests. This PR adds a less denying view there.

### Steps to test:
- run standalone datastore `sbt "webknossosDatastore/run 9090 -Dconfig.file=webknossos-datastore/conf/standalone-datastore.conf"`, navigate to http://localhost:9090
- you should see this page:
![image](https://user-images.githubusercontent.com/1390035/96586812-c2cba680-12e1-11eb-8d4c-2e054b5850f7.png)


### Issues:
- fixes #4868 

------
